docs/src/documentation/03-components/11-navigation/navigationbar/03-accessibility.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/navigationbar/accessibility/
+---
+
+## Accessibility
+
+The NavigationBar component has been designed with accessibility in mind.
+
+### Automatic Accessibility Features
+
+- Renders as a semantic `<header>` element, creating a native landmark
+- Menu button includes title text via the `openTitle` prop
+- Automatically manages focus when menu button is triggered
+
+### Keyboard Navigation
+
+- **Tab:** Moves focus to interactive elements within the navigation bar
+- **Enter/Space:** Activates buttons or links when focused
+- **Escape:** Closes active dropdown menus (when implemented)
+
+### Best Practices
+
+- Set `openTitle` to clearly describe the button navigationmenu's purpose
+- When using `transparentBgAtTop`, ensure sufficient contrast with background content
+- Use a `<nav>` element with `aria-label` inside NavigationBar to create a more descriptive navigation region
+
+### Example
+
+```jsx
+<NavigationBar id="main-navigation" onMenuOpen={handleDrawerOpen} openTitle="Open site navigation">
+  <nav aria-label="Main site navigation">
```

Comment:
As you are using Orbit components in the example, maybe you can use `<Stack as="nav" aria-label="Main site navigation">` for this one?